src/cryptography/hazmat/bindings/openssl/ocsp.py

```diff
@@ -0,0 +1,31 @@
+# This file is dual licensed under the terms of the Apache License, Version
+# 2.0, and the BSD License. See the LICENSE file in the root of this repository
+# for complete details.
+
+from __future__ import absolute_import, division, print_function
+
+INCLUDES = """
+/* Windows be crazy */
```

Comment:
The commit msg for this was "despairing windows test (this is not what we should merge)". It was made to test a Windows fix via the CI system (since that is the easiest way to test compilation across all our supported platforms) and was never intended to merge.

We may eventually need OCSP bindings but if you're not interested in them for now I'll pull them out of this PR.